Userland/Libraries/LibWeb/DOM/Document.h

```diff
@@ -445,6 +445,10 @@ class Document
     bool is_initial_about_blank() const { return m_is_initial_about_blank; }
     void set_is_initial_about_blank(bool b) { m_is_initial_about_blank = b; }
 
+    // https://html.spec.whatwg.org/multipage/dom.html#concept-document-about-base-url
+    Optional<AK::URL> about_base_url() const { return m_about_base_url; }
```

Comment:
cool, now we can pass `about base url` to document state here:
https://html.spec.whatwg.org/multipage/document-sequences.html#child-navigables:concept-document-about-base-url
and here
https://html.spec.whatwg.org/multipage/document-sequences.html#top-level-traversables:concept-document-about-base-url
and probably there are more places.
you want to fix that in this PR? :)

Reply:
Let's try not to add any more uses of the pattern of only including half of the spec comment :sweat_smile: 

Not sure how I would have found those easily without you mentioning it lol

Reply:
Unless the spec changed under us to add this concept, in which case: good catch!

Reply:
oh jeez. create_a_new_browsing_context_and_document needs some help to look like the current spec steps. But, I found the spec cross-references pretty useful for finding all the uses :+1: